monai/bundle/scripts.py

```diff
@@ -171,6 +172,10 @@ def _get_ngc_bundle_url(model_name: str, version: str) -> str:
     return f"https://api.ngc.nvidia.com/v2/models/nvidia/monaitoolkit/{model_name.lower()}/versions/{version}/zip"
 
 
+def _get_ngc_private_bundle_url(model_name: str, version: str, repo: str) -> str:
+    return f"https://api.ngc.nvidia.com/v2/{repo}/models/{model_name.lower()}/versions/{version}/zip"
```

Comment:
I'm not sure but should repo be after models?
Like 
```
f"https://api.ngc.nvidia.com/v2/models/{repo}/{model_name.lower()}/versions/{version}/zip"
```

Reply:
No, I guess the link is different with the catlog one.